cmd/podman/containers/run.go

```diff
@@ -200,7 +200,7 @@ func run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	if runOpts.Detach {
+	if runOpts.Detach && cliVals.LogDriver != define.PassthroughLogging {
```

Comment:
Shouldn't these conflict? It doesn't seem particularly sane to log to the terminal's STDOUT and STDERR when the process is no longer attached to the controlling terminal.

Reply:
the fds are passed down to the container that can still use them.

I think the only sane usage of this option is within a systemd service.

Using a tty won't be allowed by conmon as it opens to security issues: https://github.com/containers/conmon/pull/289/files#diff-26c58c117a24670c396f3e8e3e86cd6a6f134b3f02f84f8c894213c16c542d4cR165-R166

Reply:
Can we get documentation to that effect? I was very confused by what this option was for.

Also, I think conflicting with `--tty` makes a lot of sense - we won't have split stderr/stdout, and won't have raw mode set so the output will likely be garbage.

Reply:
I like Matt's suggestion

Reply:
I added the error also to Podman so we don't rely on conmon checking for it